src/Parser/Documentation.mad

```diff
@@ -1,60 +1,59 @@
 import type { Maybe } from "Maybe"
 
-import Json from "JsonParser"
+import Json from "Json/Parse"
 import { Just, Nothing } from "Maybe"
 
+
+
 export alias Instance = {
-  declaration :: String,
   constraints :: String,
-  description  :: String,
-  since        :: String,
-  example      :: String
+  declaration :: String,
+  description :: String,
+  example :: String,
+  since :: String,
 }
 
 export alias Interface = {
-  name        :: String,
-  vars        :: String,
   constraints :: String,
-  methods     :: List String,
-  description  :: String,
-  since        :: String,
-  example      :: String
+  description :: String,
+  example :: String,
+  methods :: List String,
+  name :: String,
+  since :: String,
+  vars :: String,
 }
 
 export alias Alias = {
-  name        :: String,
-  params      :: String,
   aliasedType :: String,
-  description  :: String,
-  since        :: String,
-  example      :: String
+  description :: String,
+  example :: String,
+  name :: String,
+  params :: String,
+  since :: String,
 }
 
 export alias Type = {
-  name         :: String,
-  params       :: String,
   constructors :: List String,
-  description  :: String,
-  since        :: String,
-  example      :: String
+  description :: String,
+  example :: String,
+  name :: String,
+  params :: String,
+  since :: String,
 }
 
 export alias Expression = {
-  name        :: String,
   description :: String,
-  typing      :: String,
-  since       :: String,
-  example     :: String
+  example :: String,
+  name :: String,
+  since :: String,
+  typing :: String,
 }
 
-export type Targeted a
-  = BothTargets(a, a)
-  | JSTarget(a)
-  | LLVMTarget(a)
+export type Targeted a = BothTargets(a, a) | JSTarget(a) | LLVMTarget(a) | InvalidTarget
```

Comment:
I added an `InvalidTarget` constructor for a case further down where there's an incomplete pattern for `#[Nothing, Nothing]` but this might not be needed.